db/models.py

```diff
@@ -50,3 +55,67 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
The `user` field in the `Order` model references the `User` model as a string. Ensure that the `User` model is correctly registered in the same app or specify the app label if it's in a different app to avoid potential issues.